File: sdkwork-backend-sdk-typescript/src/types/plus-table-form.ts
/** Table creation form */
export interface PlusTableForm {
  /** Table name */
  name: string;
  /** Table description */
  description?: string;
  /** Table type */
  type?: string;
  /** Column count */
  columnCount?: number;
  /** Row count */
  rowCount?: number;
  /** Last sync time */
  lastSyncTime?: string;
  /** Primary keys */
  primaryKeys?: string;
  /** Storage engine */
  engine?: string;
  /** Create SQL */
  createSql?: string;
  /** Comment */
  comment?: string;
  /** Schema ID */
  schemaId: number;
}

File: sdkwork-backend-sdk-go/types/plus_table_form.go
package types

// Table creation form
type PlusTableForm struct {
	Name string `json:"name"`
	Description string `json:"description"`
	Type string `json:"type"`
	ColumnCount int `json:"columnCount"`
	RowCount int `json:"rowCount"`
	LastSyncTime string `json:"lastSyncTime"`
	PrimaryKeys string `json:"primaryKeys"`
	Engine string `json:"engine"`
	CreateSql string `json:"createSql"`
	Comment string `json:"comment"`
	SchemaId int `json:"schemaId"`
}
